contour/src/main/kotlin/com/squareup/contour/solvers/SimpleAxisSolver.kt

```diff
@@ -83,7 +88,12 @@ internal class SimpleAxisSolver(
       if (p0.point == Point.Mid) {
         mid = p0.resolve()
       } else {
-        parent.measureSelf()
+        if (p0.point == Point.Baseline && baselineRange == Int.MIN_VALUE) {
+          parent.measureSelf()
+        }
+        else {
```

Comment:
nit: Can you run auto-format on these if-else blocks?

Reply:
Sure.